### PR TITLE
issue #1433 -- jobservice.db is not in .gitignore

### PR DESCRIPTION
**Pull request by AnkitaMalik22**

### CODE REVIEW

**Comment by kannon92** [comment]
Hello @AnkitaMalik22!  Thanks for the PR!

This looks good to me!  I'm going to wait for tests to finish before I merge.  

How did you stumble upon this project may I ask?

**Comment by AnkitaMalik22** [comment]
> 

ThankYou!actually, I found this project on the good first issue site

**Comment by kannon92** [comment]
> > 
> 
> ThankYou!actually, I found this project on the good first issue site

Can you link that site here?  I am just curious.

**Review by richscott — APPROVED** [review]

**Review by jayofdoom — APPROVED** [review]

**Pull request merged**
